docs/documentation/pages/DECKHOUSE-FAQ_RU.md

```diff
@@ -518,14 +526,21 @@ Deckhouse поддерживает работу только с Bearer token-с
 
 1. Скачайте образы модулей из их источника, описанного в виде ресурса `ModuleSource` в выделенную директорию, используя команду `dhctl mirror-modules`.
 
-   `dhctl mirror-modules` скачивает только версии модулей, доступные в каналах обновлений модуля на момент копирования.
+   `dhctl mirror-modules` скачивает только версии модулей, доступные в каналах обновлений модуля на момент копирования если не передан параметр `--modules-filter`.
 
    Следующая команда скачает образы модулей из источника, описанного в ресурсе `ModuleSource`, находящемся в файле `$HOME/module_source.yml`:
 
    ```shell
    dhctl mirror-modules -d /tmp/d8-modules -m /tmp/module_source.yml
    ```
 
+   Для загрузки только набора из определенных модулей конкретных версий, используйте параметр `--modules-filter`, передав набор необходимых модулей и их версий, разделенных символом `;`.
+   Например:
+
+   ```shell
+   dhctl mirror-modules -d /tmp/d8-modules -m /tmp/module_source.yml --modules-filter="deckhouse-admin:v1.0.0;deckhouse-admin:v1.3.3; sds-drbd:v0.0.1"
```

Comment:
--modules-filter="deckhouse-admin:v1.0.0`;`deckhouse-admin:v1.3.3`; `sds-drbd:v0.0.1" 

Do we need a space here?